docs/source/notebooks/Multimodel_simulations.ipynb

```diff
@@ -99,7 +96,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "['observed data series,filename,DIAG_NASH_SUTCLIFFE,DIAG_RMSE,\\nHYDROGRAPH_ALL,/tmp/pywps_process_xk_an7z9/input.nc,-0.117301,37.9493,\\n', 'observed data series,filename,DIAG_NASH_SUTCLIFFE,DIAG_RMSE,\\nHYDROGRAPH_ALL,/tmp/pywps_process_xk_an7z9/input.nc,-3.0132,71.9223,\\n']\n",
+      "['observed data series,filename,DIAG_NASH_SUTCLIFFE,DIAG_RMSE,\\nHYDROGRAPH_ALL,/tmp/pywps_process_vkgxnrey/Salmon-River-Near-Prince-George_meteo_daily.nc,-0.117301,37.9493,\\n', 'observed data series,filename,DIAG_NASH_SUTCLIFFE,DIAG_RMSE,\\nHYDROGRAPH_ALL,/tmp/pywps_process_vkgxnrey/Salmon-River-Near-Prince-George_meteo_daily.nc,-3.0132,71.9223,\\n']\n",
```

Comment:
I've seen this often, switch from `input.nc` to `Salmon-River-Near-Prince-George_meteo_daily.nc` and back.

Is this random?

Reply:
I think `input` filename is a default set by pywps when a file name is not provided (ie we're setting the content using raw data). But not clear why this is chaging in the nb.